src/clients/python/examples/grpc_image_client.py

```diff
@@ -64,6 +65,17 @@ def model_dtype_to_np(model_dtype):
         return np.dtype(object)
     return None
 
+def deserialize_bytes_tensor(encoded_tensor):
```

Comment:
This is already available in the utils package

Reply:
This is to be served as an example if users want to bypass our client package. Hence, I included the deserialization logic here itself. I would avoid using the client packages here.